### PR TITLE
Add config flow for integration sensor

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Adjust the docs for the addition of the configuration flow to this integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/68288
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/integration.markdown [hunk 1]
```diff
@@ -11,15 +11,42 @@ ha_quality_scale: internal
...
 ---
 
-The `integration` platform provides the [Riemann sum](https://en.wikipedia.org/wiki/Riemann_sum) of the values provided by a source sensor. The Riemann sum is an approximation of an **integral** by a finite sum. The integration sensors is updated upon changes of the **source**. Fast sampling source sensors provide better results. In this implementation, the default is the Trapezoidal method, but Left and Right methods can optionally be used.
+This integrations provides the [Riemann sum](https://en.wikipedia.org/wiki/Riemann_sum)
+of the values provided by a source sensor. The Riemann sum is an approximation
+of an **integral** by a finite sum.
 
-## Configuration
+The integration sensors are updated upon changes of the **source**. Fast
+sampling source sensors provide better results. In this implementation, the
```
Comment (emontnemery) [inline source/_integrations/integration.markdown]:
```suggestion
sampling source sensors provide more accurate results. In this implementation, the
```

File: source/_integrations/integration.markdown [hunk 2]
```diff
@@ -11,15 +11,42 @@ ha_quality_scale: internal
...
+default is the Trapezoidal method, but Left and Right methods can optionally
```
Comment (emontnemery) [inline source/_integrations/integration.markdown:25]:
```suggestion
default is the Trapezoidal rule, but Left and Right Riemann sums can optionally
```

**Review by emontnemery — DISMISSED** [review]
Thanks, @frenck 👍

File: source/_integrations/integration.markdown (hunk 2 above)
Comment (dgomes) [reply source/_integrations/integration.markdown:25]: it's not a rule... its a method https://en.wikipedia.org/wiki/Riemann_sum

Comment (frenck) [reply source/_integrations/integration.markdown:25]: 🤷

Comment (emontnemery) [reply source/_integrations/integration.markdown:25]: The article lists four methods: Left Riemann sum, Right Riemann sum, Midpoint rule and Trapezoidal rule

Comment (frenck) [reply source/_integrations/integration.markdown:25]: <https://en.wikipedia.org/wiki/Trapezoidal_rule>

**Review by emontnemery — APPROVED** [review]

File: source/_integrations/integration.markdown (hunk 2 above)
Comment (dgomes) [reply source/_integrations/integration.markdown:25]: https://en.wikipedia.org/wiki/Trapezoidal_method

🤷

**Pull request merged**